website/docs/prysm-usage/staying-up-to-date.md

```diff
@@ -189,7 +189,7 @@ User            = eth
 ExecStart       = /home/eth/prysm/prysm.sh beacon-chain --config-file=/etc/prysm/beacon-chain.yaml
 Restart         = on-failure
 TimeoutStopSec  = 900
-Environment     = USE_PRYSM_VERSION=v4.0.0
+Environment     = USE_PRYSM_VERSION=v4.1.0
```

Comment:
this will always go out of date. Should we ask people to instead find the latest version from our github?

Reply:
yeah but we do need an example, it's probably better to just leave something there, if we have a standard way of writing it maybe we can more easily update it in the docs.

Reply:
this is very frustrating, I'm about to switch to nimbus.. the downgrade will not work if you use windows scheduler..